routes/api.php

```diff
@@ -30,12 +29,12 @@
     if ($request->all()['action'] !== 'created') {
         return;
     }
-    Artisan::call('gh:change-file', [
-        'commentContent' => $request->all()['comment']['body'],
-        'filePath' => $request->all()['comment']['path'],
-        'lineNumber' => $request->all()['comment']['line'],
-        'repository' => $request->all()['pull_request']['head']['repo']['name'],
-        'owner' => $request->all()['pull_request']['head']['repo']['owner']['login'],
-        'branch' => $request->all()['pull_request']['head']['ref'],
-    ]);
+    \App\Actions\ChangeFileWithGhComment::run(
```

Comment:
adding a comment over the ChangeFileWithGhComment action with the content "//this works"